projects/packages/my-jetpack/_inc/components/product-interstitial/index.jsx

```diff
@@ -6,18 +6,23 @@ import {
 	Button,
```

Comment:
This file has turned out to be too big. May be we can split it into different components inside a directory.